### ISSUE TITLE
Running Delver using the engine compiled from source

### ISSUE THREAD

**Issue by Calinou** [Interrupt/delverengine#2]
Is it possible to run Delver using the engine compiled from this repository? This should be documented (or it should be made clear it is not possible).

**Comment by Interrupt** [comment]
Yes, that should be possible. You would need to grab the game data inside that version of the games ‘assets’ folder, and drop them in the ‘Dungeoneer/assets’ folder for this engine.

**Comment by joshuaskelly** [comment]
@Interrupt @Calinou Is this fixed? Can we close this issue?
